ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -30,6 +32,39 @@ export class AptosAccount {
     return new AptosAccount(HexString.ensure(obj.privateKeyHex).toUint8Array(), obj.address);
   }
 
+  static isValidPath = (path: string): boolean => {
+    if (
+      // eslint-disable-next-line prefer-regex-literals
+      !new RegExp('^m\\/44+\'\\/637+\'\\/[0-9]+\'\\/[0-9]+\'\\/[0-9]+\'+$').test(
```

Comment:
This doesn't look right:
1. First of all, rebase and start using double quotes
2. Let's follow the BIP44 standard, https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels. The `change` and `address_index` do not follow by a `'`. In the link, a path looks like `m / purpose' / coin_type' / account' / change / address_index`.
3. The regex has issues. `44+` would match `44444` and `637+` would match `63777777`. 44 and 637 should be an exact match. 44 is the current BIP number and 637 is Apto's coin type.

Reply:
Because it uses the [ed25519-hd-key module](https://github.com/alepop/ed25519-hd-key/blob/master/src/utils.ts), the regex check uses a `'` for all numbers. And other chains, [such as near](https://github.com/near/near-ledger-js/blob/67d5859a775290ae9bacec6bf2e2ccf6009dce30/demo/index.html#L13-L18) and [solana](https://medium.com/@josh.wolff.7/derivation-paths-on-solana-fb08d3dd09f1) are using a `'` for all numbers.